geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/RegionProvider.java

```diff
@@ -206,7 +206,7 @@ public RedisData getRedisData(RedisKey key, RedisData notFoundValue) {
 
   private RedisDataMovedException createRedisDataMovedException(RedisKey key) {
     RedisMemberInfo memberInfo = getRedisMemberInfo(key);
-    Integer slot = key.getCrc16() & (REDIS_SLOTS - 1);
+    Integer slot = key.getSlot();
```

Comment:
change "Integer" to "int" here and on RedisDataMovedException

Reply:
done